templates/includes/head.jade

```diff
@@ -51,6 +51,9 @@ head
   //- js modernizr
   script(src=contents.js.vendor['modernizr-2.8.3.min.js'].url)
 
+  //- js lazysizes
+  script(src=contents.js.vendor['lazysizes.min.js'].url)
```

Comment:
@nathakits does this need to be in the head or can we add this to `js.jade` which loads in the footer?